packages/cadl-ranch-specs/http/parameters/spread/main.tsp

```diff
@@ -260,4 +296,96 @@ namespace Alias {
   @route("/multiple-parameters/{id}")
   @put
   op spreadWithMultipleParameters(...MultipleRequestParameters): NoContentResponse;
+
+  alias BodyParameterAliasWithOptionalProps = {
+    @path id: string;
+
+    @doc("name of the Thing")
+    name: string;
+
+    @doc("optional property of the Thing")
+    color?: string;
+
+    @doc("age of the Thing")
+    age?: int32;
+
+    @header
+    `x-ms-test-header`: string;
+
+    @doc("required array")
+    items: int32[];
+
+    @doc("optional array")
+    elements?: string[];
+  };
+
+  @scenario
+  @scenarioDoc("""
+  Test case for spread alias. 
+  
+  Should  generate a model named `BodyParameterAliasWithOptionalProps`.
```

Comment:
Why should it generate a model?

Reply:
fixed

Reply:
@mcgallan  He is asking your question. Don't say "fixed" and resolve.

Reply:
I wrote this part of the doc wrong and submitted a new commit to fix it，so I just wrote a "fixed"

> @mcgallan He is asking you question. Don't say "fixed" and resolve.

I wrote this part of the doc wrong and submitted a new commit to fix it，so I just wrote a "fixed".

Reply:
You didn't understand why he asked, otherwise another one you add also "should not generated a model". 

He is asking because our logic is spread model/alias should not generate model.